src/software/thunderscope/play/playinfo_widget.py

```diff
@@ -103,5 +114,13 @@ def refresh(self):
             }
         )
 
-        self.resizeColumnsToContents()
-        self.resizeRowsToContents()
+        # TODO (#2412): Potential improvements to layout
+        self.play_table.resizeColumnsToContents()
+        self.play_table.resizeRowsToContents()
+        if referee_msg_dict:
+            p = (
+                f"REFEREE {referee_msg_dict['packetTimestamp']}: "
+                + f" {referee_msg_dict['stage']} "
```

Comment:
I think it would also be helpful to include the current referee command here as well 
```suggestion
                + f" {referee_msg_dict['stage']} - " + referee_msg_dict['command']
```